data/tests.yml

```diff
@@ -309,7 +309,8 @@
 # incorrect speed limit
 - way_id: 102917976
   description: Road with impossible speed limit
-  rust: false
```

Comment:
Ah, do you have a process for auto-formatting this file?

Reply:
I... don't know.
VS code sometimes fiddles with the yaml I write but I don't know when or why :/

It's definitely not something I would know how to integrate with the CI (without investigating)

Reply:
I reverted that line for now